.devcontainer/default/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM swiftlang/swift:nightly-main-jammy
```

Comment:
I don't have strong thoughts, but when I put this together I believe the guidance was that I should be using a "recent nightly". This should follow whatever the guidance is currently.

Reply:
@ahoppen can make the call. I guess if it is for development maybe it should stay at main

Reply:
We should use the latest `main` because the `main` branch of SourceKit-LSP might be using features from sourcekitd, that are only available in `main`. When running SourceKit-LSP tests with an older version of sourcekitd, those tests would be skipped.